packages/block-editor/src/components/block-parent-selector/index.js

```diff
@@ -60,7 +60,7 @@ export default function BlockParentSelector() {
 					onClick={ () => selectBlock( firstParentClientId ) }
 					label={ sprintf(
 						/* translators: %s: Name of the block's parent. */
-						__( 'Select parent (%s)' ),
+						__( 'Select %s' ),
```

Comment:
I don't think this text is clear enough. If you have a group block within a group block, the text 'Select Group' isn't helpful.

'Select Parent' would probably be better if it needs to be shortened.

Reply:
Looking at the UI, I don't think shortening is required as much for this button. It's on its own line so doesn't cause the toolbar to get much longer:
<img width="255" alt="Screenshot 2021-01-11 at 6 17 30 pm" src="https://user-images.githubusercontent.com/677833/104168662-5b814680-5439-11eb-8a45-29c0013e691c.png">

Some block names are also quite long compared to the word 'parent' (e.g. Navigation Link), so I don't think this change has a huge impact.

Reply:
I also agree with @talldan about not shortening the select parent button's label. Being the only one in its row, it can be left as is.

Reply:
That makes sense!